### ISSUE TITLE
[viewmodels] Sample do not build anymore on earlier iOS versions

### ISSUE THREAD

**Issue by antoinelamy** [mirego/trikot#35]
Since the introduction of `accessibilitySpeechQueueAnnouncement` [here,](https://github.com/mirego/trikot/pull/21#issuecomment-1043462439) `trikot.viewmodels` cannot build on iOS 9 and 10 anymore. Supporting an earlier version than iOS 11 is not something we want to pursue so we bump the minimum target version to iOS 11.0.
